src/lib.rs

```diff
@@ -51,18 +53,22 @@ impl Tera {
         let mut templates = BTreeMap::new();
 
         // We are parsing all the templates on instantiation
-        for entry in WalkDir::new(dir).into_iter().filter_map(|e| e.ok()) {
-            let path = entry.path();
+        for entry in glob(dir).unwrap().filter_map(|e| e.ok()) {
+            let path = entry.as_path();
             // We only care about actual files
             if path.is_file() {
+                debug!("Found file: {:?}", path);
```

Comment:
I modelled it after how hyper uses the env_logger. So, the program using tera would do the initialisation if they are interested in using debugging. Otherwise it should have no effect, but I agree that the dependency should be not just be for dev, I'll refactor.